src/OSPSuite.R/Services/DataRepositoryTask.cs

```diff
@@ -28,5 +35,34 @@ public void SaveDataRepository(DataRepository dataRepository, string fileName)
       {
          _pkmlPersistor.SaveToPKML(dataRepository, fileName);
       }
+
+      public DataColumn GetErrorColumn(DataColumn column)
+      {
+         if (column.ContainsRelatedColumn(AuxiliaryType.ArithmeticStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.ArithmeticStdDev);
+
+         if (column.ContainsRelatedColumn(AuxiliaryType.GeometricStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.GeometricStdDev);
+
+         return null;
+      }
+
+      public DataColumn AddErrorColumn(DataColumn column, string name = "yError", string errorType = null)
```

Comment:
I have foud that optional parameters are an issue wth rCLR. Are you sure this will work?

Reply:
Yes you are right.

@abdelr Then I think let's just make the errorType mandatory with no default value.

Reply:
They are mandatory now